content/documentation/pages/1-installation/1-local/3-manual.md

```diff
@@ -10,28 +10,30 @@ If Docker does not suit your needs, you can manually install the parts you need
 
 ## Download server jars
 
-1.  Download the Spring Cloud Data Flow Server and shell by using the following command:
+1.  Download the Spring Cloud Data Flow Server and shell by using the following commands:
 
         wget https://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-server/{%dataflow-version%}/spring-cloud-dataflow-server-{%dataflow-version%}.jar
```

Comment:
%dataflow-version% is not resolving to version number

Reply:
Yeah, I brought it up with @oodamien on a slack thread earlier today. It seems like a bug.